platforms/documentation/docs/src/snippets/plugins/dependenciesBlock/groovy/buildSrc/src/main/groovy/com.example.custom-dependencies.gradle

```diff
@@ -0,0 +1,9 @@
+def example = extensions.create("example", com.example.ExampleExtension)
```

Comment:
🤔 I don't like how this example code requires 2 languages (Groovy and Java).  Seems to make it more complex than it actually it.  I'd consider writing the convention plugin in Java and just going all Java.

As we're Kotlin first, alternately, this Groovy plugin should arguably be replaced by Kotlin.

Reply:
I replaced the implementation of the plugin with a pure Java one.  I left the precompiled script plugin since it's invisible to the snippet and keeps the buildSrc build simpler.